examples/medplum-demo-bots/src/deduplication/README.md

```diff
@@ -0,0 +1,95 @@
+# Patient Deduplication Demo
+
+This folder contains a reference implementation for a simple, human-in-the-loop patient deduplication pipeline, based on the guidance in [Patient Deduplication Architectures](https://www.medplum.com/docs/fhir-datastore/patient-deduplication#architecture-overview).
```

Comment:
Should also include a link to this bot from the https://www.medplum.com/docs/fhir-datastore/patient-deduplication#architecture-overview as well.